src/components/include/application_manager/application_manager.h

```diff
@@ -585,6 +586,27 @@ class ApplicationManager {
 
   virtual void RemoveAppFromTTSGlobalPropertiesList(const uint32_t app_id) = 0;
 
+  /**
+   * @brief Resets application's global properties to default values
+   * @param container with global properties IDs to reset
+   * @param id of app which properties to reset
+   * @return struct with flags indicating success global properties reset
+   */
+  virtual ResetGlobalPropertiesResult ResetGlobalProperties(
+      const smart_objects::SmartObject& global_properties_ids,
+      const uint32_t app_id) = 0;
+
+  /**
+   * @brief Resets all application's global properties to default values
+   * returning struct that indicates which properties have been
+   * successfully reset.
+   * @param container with global properties to reset
```

Comment:
there is no such param

Reply:
Fixed in 1c1f20898109fd9781451234e4032784ef97c4f6